storybookjs/storybook
# Added documentation related to issue #1820

Issue: #1820 

## What I did
Added documentation showing how to provide `createNodeMock` to Storyshots as requested in #1820.

## How to test

Is this testable with jest or storyshots?
No

Does this need a new example in the kitchen sink apps?
Not sure. There was an example here: https://github.com/storybooks/storybook/blob/b915b5439786e0edb17d7f5ab404bba9f7919381/examples/test-cra/src/storyshots.test.js#L14-L16

Does this need an update to the documentation?
Yes

If your answer is yes to any of these, please make sure to include it in your PR.
